sdk/samples/iot/paho_iot_hub_pnp_sample.c

```diff
@@ -852,18 +852,19 @@ static az_result invoke_getMaxMinReport(
       command_start_time_value_buffer,
       sizeof(command_start_time_value_buffer),
       &incoming_since_value_len));
-  az_span start_time_span
-      = az_span_create((uint8_t*)command_start_time_value_buffer, incoming_since_value_len);
-
-  IOT_SAMPLE_LOG_AZ_SPAN("start time:", start_time_span);
 
   // Set the response payload to error if the "since" value was empty.
-  if (az_span_ptr(start_time_span) == NULL)
```

Comment:
@momuno - This would have never been true previously because `command_start_time_value_buffer` is not null. FYI.

Reply:
oh of course.  good catch.

Reply:
Also, as a recommendation, avoid using `az_span_ptr(...)` where possible, particularly when making just size-based decisions. Only use it when you need to access the contents of the span buffer either for reading or writing.